src/lib/types.ts

```diff
@@ -117,6 +117,11 @@ export interface CommandResult {
   message: string | undefined;
 }
 
+export interface CompareAndUpdateBranchesResponse {
```

Comment:
this type is not likely to ever be used widely, would be best next to the function?